File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/DailyActivityTestsFormViewModel.cs
//$C422C148
using System;
using System.Linq;
using Core.Models;
using Infrastructure.Filters;
using Infrastructure.Models;
using TTInstanceManagement;
using TTObjectClasses;
using TTDefinitionManagement;
using TTUtils;
using Core.Controllers;

namespace Core.Controllers
{
    public partial class DailyActivityTestsFormServiceController
    {
    }
}

namespace Core.Models
{
    public partial class DailyActivityTestsFormViewModel : BaseAdditionalInfoFormViewModel
    {
        public override BaseAdditionalInfo AddViewModelToContext(TTObjectContext objectContext)
        {
            var dailyActivityTestsFormService = new DailyActivityTestsFormServiceController();
            dailyActivityTestsFormService.DailyActivityTestsFormInternal(this, objectContext);
            return (DailyActivityTestsForm)objectContext.GetObject(this._DailyActivityTestsForm.ObjectID, "DailyActivityTestsForm");
        }
    }
}

File: AtlasWeb/Core/Modules/Tibbi_Surec/Fizyoterapi_Modulu/DailyActivityTestsFormViewModel.ts
//$43B39B65
import { BaseViewModel } from "NebulaClient/Model/BaseViewModel";
import { DailyActivityTestsForm } from "NebulaClient/Model/AtlasClientModel";
import { Type } from 'NebulaClient/ClassTransformer';

export class DailyActivityTestsFormViewModel extends BaseViewModel {
    protected __type__: string = 'Core.Models.DailyActivityTestsFormViewModel, Core';
    @Type(() => DailyActivityTestsForm)
    public _DailyActivityTestsForm: DailyActivityTestsForm = new DailyActivityTestsForm();
}
